sklearn/model_selection/_search.py

```diff
@@ -556,11 +559,14 @@ def _fit(self, X, y, labels, parameter_iterable):
         stds = np.sqrt(np.average((test_scores - means[:, np.newaxis]) ** 2,
                                   axis=1, weights=weights))
 
+        mean_time = np.average(scoring_time, axis=1)
+
         results = dict()
         for split_i in range(n_splits):
             results["test_split%d_score" % split_i] = test_scores[:, split_i]
         results["test_mean_score"] = means
         results["test_std_score"] = stds
+        results["test_mean_time"] = mean_time
```

Comment:
But if we don't have individual timings, we have no sense of whether some runs were outlying...?